src/cli/llm/infer_type_name.rs

```diff
@@ -76,20 +84,43 @@ impl InferTypeName {
         Self { wizard: Wizard::new(model, secret) }
     }
 
+    /// Determines if a type name is automatically generated.
+    fn is_auto_generated(type_name: &str) -> bool {
```

Comment:
this will skip this autogenerated gRPC names, add check to ensure gRPC names are considered as auto generated